src/app/routes/broker-ui/connector-page/connector-cards/connector-cards.component.html

```diff
@@ -1,10 +1,28 @@
 <mat-card *ngFor="let connector of connectors" class="asset-card">
-  <mat-card-header>
-    <mat-icon class="mat-icon-[40px]" mat-card-avatar>link</mat-icon>
-    <mat-card-title>
-      {{ connector.endpoint }}
-    </mat-card-title>
-    <mat-card-subtitle>EDC Connector</mat-card-subtitle>
+  <mat-card-header class="header-content">
+    <div class="icon-wrapper">
+      <mat-icon class="main-icon" mat-card-avatar>link</mat-icon>
+      <div class="small-icon-wrapper">
+        <mat-icon
+          *ngIf="connector.onlineStatus === 'ONLINE'"
+          class="small-icon"
+          [ngStyle]="{color: 'green'}"
+          >check_circle</mat-icon
+        >
+        <mat-icon
+          *ngIf="connector.onlineStatus === 'OFFLINE'"
+          class="small-icon"
+          [ngStyle]="{color: 'red'}"
```

Comment:
we should maybe externalize these colours, since they are used in multiple places, for example, in the style.scss

.broker-online-status-online, .broker-online-status-offline, .broker-online-status-dead

Reply:
Added external classes in styles.scss